iphone/Classes/TiUIWebView.m

```diff
@@ -715,8 +715,8 @@ -(CGFloat)contentHeightForWidth:(CGFloat)value
     CGRect oldBounds = [[self webview] bounds];
     BOOL oldVal = webview.scalesPageToFit;
     [webview setScalesPageToFit:NO];
-    [webview setBounds:CGRectMake(0, 0, 10, 1)];
-    CGFloat ret = [webview sizeThatFits:CGSizeMake(10, 1)].height;
+    [webview setBounds:CGRectMake(0, 0, value, 1)];
```

Comment:
Where did the static 10 come from? Wondering why that particular value was used (and is also still used in `contentWidthForWidth :`).